bam.lua

```diff
@@ -177,6 +177,8 @@ function GenerateLinuxSettings(settings, conf, arch, compiler)
 		settings.link.flags:Add("-m64")
 	elseif arch == "armv7l" then
 		-- arm 32 bit
+	elseif arch == "aarch64" then
```

Comment:
This isn't working cause bam don't support it (https://github.com/matricks/bam/blob/master/src/platform.h#L131).